docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,49 +1,58 @@
 [[run-digger-installer]]
-== Run Digger Installer
+=== Run Digger Installer
 
-.To run the Digger installer, follow these steps:
 
-. Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo^]
+. Clone the https://github.com/aerogear/aerogear-digger-installer[Digger Installer GitHub repo^]
+
+. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure. https://github.com/aerogear/aerogear-digger-installer/blob/master/inventory-sample[Sample Ansible Inventory file^].
 
-. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure
 +
-NOTE: For a full list of variables that can be configured via the inventory file see the link:#installer-variables[Installer Variables] section.
+NOTE: For a full list of variables that can be configured via the inventory file refer to <<installer-variables>>
 
-. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal.:
+. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal:
++
+NOTE: Review https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^].
 
-** Command to install digger without a macOS server
+** Command to install Digger without a macOS server
 +
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=osx-provision
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx
 ----
 
-** Command to install digger with a macOS server
+** Command to install Digger with a macOS server
 +
 ----
 ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 ----
 
-. Carry out manual steps during the installation
+** Roles can be run individually or in batch with flag --tags=<role_tag_1>,...,<role_tag_x>
```

Comment:
I'm not sure if I remember this correctly but I thought we decided not to include the information about how to skip/run particular roles. There is a fix required on the tags first. If the fix is done then we can include it here. I would suggest checking that.

Reply:
@laurafitzgerald I can't recall talking about this unfortunately. Could you verify this info?

Reply:
@pmacko1 I will ask in the channgel

Reply:
Yep it's related to this JIRA: https://issues.jboss.org/browse/AGDIGGER-141

Reply:
from rocketchat: https://issues.jboss.org/browse/AGDIGGER-260
https://issues.jboss.org/browse/AGDIGGER-208
https://issues.jboss.org/browse/AGDIGGER-141
several tags should not be skipped/used individually

Reply:
@pmacko1 could we remove this section for the moment. If you copy the commit elsewhere and leave it on a branch it can be linked in the JIRAs to resolve the issue and will be ready for merging once it is available.

Reply:
just commented out

resolved